src/lib/hooks/useChart.ts

```diff
@@ -20,7 +20,9 @@ const useChart = () => {
     getCharts();
   }, []);
 
-  return { charts, start, end };
+  const chartDistrict = [...new Set(charts.map((chart) => chart.id))];
```

Comment:
```suggestion
  const chartDistrict =  Array.from(new Set(charts.map((chart) => chart.id))]);
```
이렇게 하면 열거형을 쓰지 않아도 됩니다! 하지만 열거형이 가독성은 더 좋아보이네요..ㅎㅎ..

Reply:
이거 코드리뷰 하다가 급히 수정했는데 반영 안되었을까요?!
감사합니다 소연님 😊

Reply:
앗,, 변경전에 리뷰 작성해놔서 몰랐네요😅